test/Dockerfile.python-3.10

```diff
@@ -27,11 +27,12 @@ RUN pip install \
   coloredlogs \
   "coverage[toml] >= 5.0" \
   lxml \
+  pytest \
   pystemd \
   pyyaml \
   'setuptools < 58'
 
 # Avoid the wheel on PyPi for nose, because it does not work on Python 3.10.
 # An installation from source does work, though, if setuptools<58 exists.
 # Cf. https://github.com/nose-devs/nose/issues/1099
-RUN pip install nose --no-binary :all:
+# RUN pip install nose --no-binary :all:
```

Comment:
If we don't need it anymore, then it should be deleted, not just commented out. Also note that nose was the reason for the requirement of `setuptools<58` (as the comment explains), so we can remove that as well.